docs/conflicts.md

```diff
@@ -50,61 +50,117 @@ For information about how conflicts are handled in the working copy, see
 
 ## Conflict markers
 
+Conflicts happen when Jujutsu can't figure out how to merge different changes
+made to the same file. For instance, this can happen if two people are working
+on the same file and make different changes to the same part of the file, and
+then their commits are merged together with `jj new` (or one is rebased onto the
+other with `jj rebase`).
```

Comment:
I feel like this paragraph belongs at the start of "Introduction" (before what's currently on line 6). What do you think?

Reply:
Makes sense to me! I moved it.